packages/at_client/lib/src/transformer/response_transformer/get_response_transformer.dart

```diff
@@ -35,39 +36,66 @@ class GetResponseTransformer
       atValue.metadata = metadata;
       tuple.one.metadata = metadata!;
     }
-
     // For public and cached public keys, data is not encrypted.
+    if (_isKeyPublic(decodedResponse['key'])) {
+      return _handlePublicData(atValue, tuple);
+    }
+    decryptionManager ??= AtKeyDecryptionManager(_atClient);
+    var decryptionService =
+        decryptionManager!.get(tuple.one, _atClient.getCurrentAtSign()!);
     // Decrypt the data, for other keys
-    if (!(decodedResponse['key'].startsWith('public:')) &&
-        !(decodedResponse['key'].startsWith('cached:public:'))) {
-      var decryptionService = AtKeyDecryptionManager(_atClient)
-          .get(tuple.one, _atClient.getCurrentAtSign()!);
+    // For new encrypted data after AtClient v3.2.1, isEncrypted will be true by default
```

Comment:
What does "by default" mean? Is it more accurate to say that "isEncrypted will be set to true when data is encrypted"?

Reply:
isEncrypted will be set to true for self/shared keys, if caller doesn't set anything in PutRequestionOptions. I will change the wording.